File: my-folder/1528-kids-with-the-greatest-number-of-candies/solution.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max_num = max(candies)

        res = [False] * len(candies)

        for i in range(len(candies)):
            if candies[i] + extraCandies >= max_num:
                res[i] = True
        
        return res

File: my-folder/1528-kids-with-the-greatest-number-of-candies/solution.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int max = 0;
        for (int candy : candies) {
            max = Math.max(candy, max);
        }
        System.out.println(max);
        ArrayList<Boolean> res = new ArrayList<Boolean>();
        for (int i = 0; i < candies.length; i++) {
            if (candies[i] + extraCandies >= max) {
                res.add(true);
            }
            else {
                res.add(false);
            }
        }
        return res;
    }
}
